src/settings/user.rs

```diff
@@ -93,7 +79,7 @@ impl UserConfiguration {
                 .into_iter()
                 .map(|per_file_ignore| {
                     (
-                        Exclusion::from_file_pattern(per_file_ignore.pattern),
+                        Exclusion::from_file_pattern(&per_file_ignore.pattern),
```

Comment:
I think we should remove `Exclusion` and just inline the string, now that `basename` and `absolute` are identical.

Reply:
cc32661 re-split basename and absolute, we should continue to use `Exclusions`